extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -144,24 +151,45 @@ class="jetpack-podcast-player <?php echo esc_attr( $player_classes_name ); ?>"
 				)
 			);
 			?>
+			<?php if ( $is_amp ) : ?>
+			<amp-selector
+				layout="container"
+				keyboard-select-mode="select"
+				class="jetpack-podcast-player__tracks"
+				on="select: AMP.setState({
+					currentTrack: event.targetOption,
+				})"
+			>
+			<?php else : ?>
 			<ol class="jetpack-podcast-player__tracks">
+			<?php endif; ?>
 				<?php foreach ( $player_data['tracks'] as $track_index => $attachment ) : ?>
 					<?php
 					render(
 						'playlist-track',
 						array(
+							'index'            => $track_index,
 							'is_active'        => 0 === $track_index,
 							'attachment'       => $attachment,
 							'primary_colors'   => $primary_colors,
 							'secondary_colors' => $secondary_colors,
+							'is_amp'           => $is_amp,
 						)
 					);
 					?>
```

Comment:
How about extracting this to a shared template file? It could then be reused for separate AMP and Standards files.